javascript/chronometer.js

```diff
@@ -1,26 +1,72 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.current_seconds = 0;
+    this.interval_seconds = 0;
+
+    this.current_milliseconds = 0;
+    this.interval_milliseconds = 0;
   }
-  startClick(callback) {
-    // ... your code goes here
+
+  startTimer(callback) {
+    this.startMilliseconds(callback);
+    this.startSeconds(callback);
   }
+
+  startSeconds(callback) {
+    this.interval_seconds = setInterval(() => {
+      this.current_seconds++;
+      callback();
+    }, 1000);
+  }
+
+  startMilliseconds(callback) {
+    this.interval_milliseconds = setInterval(() => {
+      this.current_milliseconds++;
+      callback();
+    }, 1);
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.current_seconds / 60);
   }
+
   getSeconds() {
-    // ... your code goes here
+    return this.current_seconds - this.getMinutes() * 60;
+  }
+
+  getMilliseconds() {
+    return this.current_milliseconds % 100;
   }
-  twoDigitsNumber() {
-    // ... your code goes here
+
+  twoDigitsNumber(current_time) {
+    if (current_time < 10) {
+      return "0" + String(current_time);
```

Comment:
current_time will be automatically converted to a string here when you concatenate it with another string. so no need for the String( )